manifest.php

```diff
@@ -31,7 +31,7 @@
     'label' => 'Test core extension',
     'description' => 'TAO Tests extension contains the abstraction of the test-runners, but requires an implementation in order to be able to run tests',
     'license' => 'GPL-2.0',
-    'version' => '13.1.0',
+    'version' => '13.1.1',
```

Comment:
could you please make it a feature version ?
and also enfore `tao >= 38.5.0` so all libraries versions are enforced

Reply:
fixed